packages/js/product-editor/src/blocks/generic/pricing/edit.tsx

```diff
@@ -21,13 +21,14 @@ import { useCurrencyInputProps } from '../../../hooks/use-currency-input-props';
 import { PricingBlockAttributes } from './types';
 import { ProductEditorBlockEditProps } from '../../../types';
 import useProductEntityProp from '../../../hooks/use-product-entity-prop';
+import { Label } from '../../../components/label/label';
 
 export function Edit( {
 	attributes,
 	context: { postType },
 }: ProductEditorBlockEditProps< PricingBlockAttributes > ) {
 	const blockProps = useWooBlockProps( attributes );
-	const { property, label, help, disabled } = attributes;
+	const { property, label, help, disabled, tooltip } = attributes;
```

Comment:
I think you could remove the `labelToShow` constant by setting the default value here:

```suggestion
	const { property, label = __( 'Price', 'woocommerce' ), help, disabled, tooltip } = attributes;
```

Reply:
Good call! I did it in the commit https://github.com/woocommerce/woocommerce/pull/43144/commits/f41aaed8b302a46921f16aa5979c250bd9a34f41